dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/consent/ConsentDetails.java

```diff
@@ -0,0 +1,96 @@
+package edu.harvard.iq.dataverse.persistence.consent;
+
+import edu.harvard.iq.dataverse.persistence.config.LocaleConverter;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import java.util.Locale;
+
+@Entity
+public class ConsentDetails {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(nullable = false, name = "consent_id")
+    private Consent parent;
```

Comment:
I'd rather call it something like simple `consent` and have getter `getConsent` or more commonly used in the app `owner`. I feel like `parent` is not verbose enough.